modules/schedulers/src/test/java/org/mule/test/modules/schedulers/cron/CronsSchedulerTestCase.java

```diff
@@ -15,7 +15,7 @@
 import org.mule.runtime.api.exception.MuleException;
```

Comment:
Not sure when this was committed, but it should be singular. `CronsSchedulerTestCase.java` -> `CronSchedulerTestCase.java`